src/components/activity-page/section.scss

```diff
@@ -8,12 +8,28 @@
   margin: 10px;
   position: relative;
   &.responsive {
-    width: 100%;
+    width: calc(100% - 20px);
```

Comment:
Why is the width reduced here? Playing around with it in browser a 100% width looks better than 20px less. But maybe I'm missing something?